deploy/kubernetes/overlays/local/ingress-nginx/README.md

```diff
@@ -0,0 +1,21 @@
+# NGINX Configs
+
+Installing baremetal NGINX requires running the following commands according to the [directions](https://kubernetes.github.io/ingress-nginx/deploy/) on the nginx site.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.28.0/deploy/static/mandatory.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.28.0/deploy/static/provider/cloud-generic.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.28.0/deploy/static/provider/baremetal/service-nodeport.yaml
+```
+
+The directories below contain the contents of these three configs
+1. Split out into their component yaml files.
+2. Added `kustomization.yaml` files tying them together.
+3. Removed conflicting resources.
+
+The kustomize dependency graph looks like so:
+```
+overlays\local -> overlays\local\ingress-nginx\baremetal
+overlays\local\ingress-nginx\baremetal -> overlays\local\ingress-nginx\cloudgeneric
+overlays\local\ingress-nginx\cloudgeneric -> overlays\local\ingress-nginx\static
+```
```

Comment:
Just curious: Under which circumstances would one need to modify these files?

As discussed offline, let's see how this grows. It's certainly very valuable to have travis validate this in CI, but given the potential divergence between the deployment and this `local test`, we might not be validating much that overlaps with a live deployment. Let's keep in mind a full E2E test deployment.

Reply:
The only time we would want to update these files it to keep up-to-date with upstream changes. 

Agreed that the local test is becoming less useful, but let's keep an eye on it.